### DISCUSSION TITLE
How do you use Anki to stay in touch with others?

### DISCUSSION THREAD

**Discussion by tooskar** [BryanJenksCommunity/FAQ#169] [Q&A] [answered]
In your videos you mention that you require some efforts not to ignore those you value but don't see often, and that you use Anki to remedy that. I don't remember which video it was, so I'm sorry if I'm misquoting you.
I've been in the same boat my whole life and have come to use apps to remind me of a person every X days or months. How do you emloy SRS algorithm for that? Do you have any other software that help you to connect with people?

PS I admire how relentlessly you keep at bay your ADHD using meds, software, etc. I try to do the same with my disorder of different name and different symtoms related mainly to social ineptness. Still I find your videos useful, mainly Obsidian related. Thanks for posting them!

**Comment by tallguyjenks** [comment, marked as answer]
i would just make cards and review them and it reminded me to reach out but i do not really find this effective anymore as the SRS algorithm is not really great for this type of use case. Now i may just use CRM software like monicahq. still looking for good solutions

glad you're enjoying the content :)

**Reply by tooskar** [reply]
That's not the answer I was initially hoping for. However, I got even better tip. Monica looks promising. Thanks!
